docs/source/user-guide/getting-started.rst

```diff
@@ -2,306 +2,137 @@
-used when you downloaded and installed Anaconda. If you want to use a different
-version of Python, for example Python 3.5, simply create a new environment and
-specify the version of Python that you want.
-
-#. Create a new environment named "snakes" that contains Python 3.9:
-
-   .. code::
-
-      conda create --name snakes python=3.9
-
-   When conda asks if you want to proceed, type "y" and press Enter.
-
-#. Activate the new environment:
-
-   * Windows: ``conda activate snakes``
-   * macOS and Linux: ``conda activate snakes``
-
-   .. note::
-      ``conda activate`` only works on conda 4.6 and later versions.
-
-   For conda versions prior to 4.6, type:
-
-   * Windows: ``activate snakes``
-   * macOS and Linux: ``source activate snakes``
-
-#. Verify that the snakes environment has been added and is active:
-
-   .. code::
-
-      conda info --envs
-
-   Conda displays the list of all environments with an asterisk (*)
-   after the name of the active environment:
-
-   .. code::
-
-     # conda environments:
-     #
-     base                     /home/username/anaconda3
-     snakes                *  /home/username/anaconda3/envs/snakes
-     snowflakes               /home/username/anaconda3/envs/snowflakes
-
-   The active environment is also displayed in front of your prompt in
-   (parentheses) or [brackets] like this:
+.. tip::
+   The active environment is the one with an asterisk (*).
 
-   .. code::
+To change your current environment back to the default ``base``::
 
-     (snakes) $
+   conda activate
 
-#. Verify which version of Python is in your current
-   environment:
+.. tip::
+    When the environment is deactivated, its name is no longer shown in your prompt, and the asterisk (*) returns to ``base``. To verify, you can repeat the  ``conda info --envs`` command.
 
-   .. code::
+Installing packages
+===================
 
-      python --version
+You can also install packages into a previously created environment. To do this, you first need to activate the environment. This changes the environment shown in your shell from ``(base)`` to the name of the environment. **Don’t install packages into your base environment.**::
 
-#. Deactivate the snakes environment and return to base environment:
-   ``conda activate``
+   conda activate myenvironment
+   conda install matplotlib
 
-   .. note::
-      For versions prior to conda 4.6, use:
+For more information on searching for and installing packages, see :doc:`Managing packages <tasks/manage-pkgs>`.
 
-        * Windows:  ``activate``
-        * macOS, Linux: ``source activate``
+Specifying channels
+===================
 
+Channels are locations (on your own computer or elsewhere on the Internet) where packages are stored. By default, conda searches for packages in its :ref:`default channels <default-channels>`.
 
-.. _managing-pkgs:
+If a package you want is somewhere else, such as conda-forge, you can manually specify the channel when installing the package::
```

Comment:
```suggestion
If a package you want is located in another channel, such as conda-forge, you can manually specify the channel when installing the package::
```

Reply:
Maybe it would be nice to add a "Tip" admonition about anaconda.org for finding more packages and channels?